assets/src/vo/date-time/datetime.js

```diff
@@ -340,18 +361,14 @@ export default class DateTime {
 			! isUndefined( momentInstance.tz() ) &&
 			momentInstance.tz() !== 'UTC' ?
 			new this(
-				...this[ privateMethods.normalizeArguments ](
-					momentInstance.toISOString(),
-					momentInstance.tz(),
-					momentInstance.locale()
-				)
```

Comment:
the sole purpose of `normalizeArguments()` was to swap the order of the properties depending on whether you were creating an instance of `Datetime` or `ServerDateTime`

Reply:
I hate overkill-ing the simple needs.